### PR TITLE
Cosmetic streamlining

### PR DESCRIPTION
**Pull request by rfranzke**
**How to categorize this PR?**

/area dev-productivity
/kind cleanup

**What this PR does / why we need it**:
This PR is about a few cosmetic streamlining for better consistency across the code base.

**Release note**:

```other operator
NONE
```

### CODE REVIEW

**Comment by shafeeqes** [comment]
Can we tackle https://github.com/gardener/gardener/pull/7281#discussion_r1069229637, also as part of this PR? WDYT?

**Comment by rfranzke** [comment]
I don't think there is the necessity for this to be put into this PR (which mostly is about naming).

File: extensions/pkg/util/secret/manager/manager_test.go [hunk 1]
```diff
@@ -173,7 +173,7 @@ var _ = Describe("SecretsManager Extension Utils", func() {
 			})
 
 			JustBeforeEach(func() {
-				By("creating old CA secrets")
+				By("Create old CA secrets")
```
Comment (acumino) [inline extensions/pkg/util/secret/manager/manager_test.go:176]: There are still files remaining, where By("") has been used in which string's first letter is not capitalized.
You can find such occurrences using this `By\("[a-z]` regex.

File: extensions/pkg/controller/backupentry/backupentry_suite_test.go [hunk 2]
```diff
@@ -21,7 +21,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-func TestBackupentry(t *testing.T) {
+func TestBackupEntry(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "Backupentry Suite")
+	RunSpecs(t, "Extensions Controller BackupEntry Suite")
```
Comment (acumino) [inline extensions/pkg/controller/backupentry/backupentry_suite_test.go:26]: There are a few occurrences left in extensions that are not named as per this convention.
You can find such occurrences using `RunSpecs\(t, "[^E]` regex.

Comment (rfranzke) [reply extensions/pkg/controller/backupentry/backupentry_suite_test.go:26]: Thanks!

File: extensions/pkg/util/secret/manager/manager_test.go (hunk 1 above)
Comment (rfranzke) [reply extensions/pkg/util/secret/manager/manager_test.go:176]: Thanks!

**Review by acumino — COMMENTED** [review]
Thanks!
/lgtm
/approve

**Pull request merged**